lms/static/js/spec/search/search_spec.js

```diff
@@ -611,19 +611,7 @@ define([
                 $('.search-button').trigger('click');
                 // cancel search
                 $('.cancel-button').trigger('click');
-                AjaxHelpers.respondWithJson(requests, {
-                    total: 1337,
-                    access_denied_count: 12,
-                    results: [{
-                        data: {
-                            location: ['section', 'subsection', 'unit'],
-                            url: '/some/url/to/content',
-                            content_type: 'text',
-                            excerpt: 'this is a short excerpt',
-                            course_name: ''
-                        }
-                    }]
-                });
+                AjaxHelpers.skipResetRequest(requests);
```

Comment:
FWIW I did go to the skipResetRequest function definition to see what it did but didn't fully understand it. Maybe if there was a comment there saying that it's typically used after hitting a cancel button which should reset the request (like you explained above) that would be sufficient.